README.md

```diff
@@ -20,18 +20,12 @@ var through = require('through2')
 
 var str = hstring(memdb())
 
-str.insert(null, 'H', function (err, op) {
-  str.insert(op.pos, 'e', function (err, op2) {
-    str.insert(op2.pos, 'l', function (err, op3) {
-      str.insert(op3.pos, 'l', function (err, op4) {
-        str.insert(op4.pos, 'o')
-        str.insert(op2.pos, 'y')  // two inserts at 'op2.pos'!
-      })
-    })
-  })
+str.insert(null, 'Helo', function (err, ops) {
+  // ops is an array of four insert ops
+  str.insert(ops[2].pos, 'l')
 })
 
-str.createStringStream()
+str.createReadStream()
```

Comment:
Sorry for the confusion, this is old code. `createReadStream` returns a readable stream of **hyperlog** operations, not in-order characters. You'll want to use `hstring.text()` here to get the string literal of the hyper-string, and drop the `through`/piping business. Just

```js
str.text(function (text) {
  console.log(text)
})
```